discord/user.py

```diff
@@ -707,6 +707,17 @@ def dm_channel(self):
         """
         return self._state._get_private_channel_by_user(self.id)
 
+    @property
+    def mutual_guilds(self):
+        """List[:class:`Guild`]: The guilds that the user shares with the client.
+
+        .. note::
+
+            This can only return mutual guilds within the client's internal cache.
+        .. versionadded:: 1.7
```

Comment:
```suggestion
        .. note::

            This will only return mutual guilds within the client's internal cache.

        .. versionadded:: 1.7
```